tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "T_eff = 4780 * u.K  # temperature of hatp11 [K]\n",
+    "\n",
+    "# Use synphot.models and synphot.spectrum module to generate a blackbody spectrum\n",
+    "photlam_hatp11 = SourceSpectrum(BlackBody1D, temperature=T_eff) # in photon s-1 cm^-2 AA^-1\n",
+    "\n",
+    "# Compute the flux at Earth from the luminosity \n",
+    "R = 0.683 * u.R_sun  # radius in solar units\n",
+    "d = (123 * u.lyr).to(u.R_sun)  # distance to HATP11 in solar radii\n",
+    "photlam_hatp11 = photlam_hatp11 * (R / d) ** 2\n",
+    "\n",
+    "photlam_hatp11.plot(left=3000, right=10000)"
```

Comment:
When plotting, user usually connects more with flux unit like FLAM or Jy. You can use `flux_unit` to set the displayed flux unit, as documented in [synphot.spectrum.SourceSpectrum.plot](https://synphot.readthedocs.io/en/latest/api/synphot.spectrum.SourceSpectrum.html#synphot.spectrum.SourceSpectrum.plot) .